src/sidebar/store/modules/selection.js

```diff
@@ -92,29 +92,40 @@ const setTab = (selectedTab, newTab) => {
 
 function init(settings) {
   return {
-    // An array of annotation `$tag`s representing annotations that are focused
-    focusedAnnotations: [],
-
-    // Contains a map of annotation id:true pairs.
-    selectedAnnotationMap: initialSelection(settings),
-
-    // Map of annotation IDs to expanded/collapsed state. For annotations not
-    // present in the map, the default state is used which depends on whether
-    // the annotation is a top-level annotation or a reply, whether it is
-    // selected and whether it matches the current filter.
+    /**
+     * The following objects map annotation identifiers to a boolean
+     * (typically `true`). They are objects (i.e. instead of Arrays) for two
+     * reasons:
+     * - Allows explicit setting of `false`
+     * - Prevents duplicate entries for a single annotation
+     */
+
+    // A set of annotations that are currently "focused" — hovered over in
+    // the UI, e.g.
```

Comment:
Was there a comment missing after `e.g.`?

Reply:
No, but my grammar may be a little hard to parse; I'll rewrite as `e.g. hovered over in the UI`